packages/visual-editor/src/components/ReadOnlyEditorBlock.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useMemo, useRef } from 'react';
```

Comment:
[8] Mostly the same as the blocks for the delivery mode just getting the information differently as we are already in the editor mode land. 

This might be abstracted and be shared with the delivery mode components ?